draft-howard-rats-coserv.md

```diff
@@ -672,6 +672,81 @@ If possible, TBD1 and TBD2 should be assigned in the 256..9999 range.
 # Acknowledgments
 {:numbered="false"}
 
-TODO acknowledge.
+The participants to the "Staircase meeting" at FOSDEM 25:
```

Comment:
This should be "in" or "at" or "of" rather than "to".

Reply:
Fixed in [dd611a6](https://github.com/rats-endorsements-distribution/draft-howard-rats-coserv/pull/53/commits/dd611a63ef6417d55e26482de246144352defbd0), thanks!